Source/Core/Core/Boot/Boot.h

```diff
@@ -32,6 +32,14 @@ struct RegionSetting
   std::string code;
 };
 
+// CRC32 hashes of the IPL file, obtained from Redump
+constexpr u32 NTSC_v1_0 = 0x6DAC1F2A;
```

Comment:
Why were these moved out here? These can be kept within the cpp file.

Reply:
Good point